testlapack/dlasq1.go

```diff
@@ -0,0 +1,109 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+func printFortranArray(z []float64, name string) {
```

Comment:
These fortran output mimics could go in there own file with more documentation about why they exist.